src/psyclone/psyir/symbols/symboltable.py

```diff
@@ -43,7 +43,7 @@
 import six
 from psyclone.configuration import Config
 from psyclone.psyir.symbols import Symbol, DataSymbol, GlobalInterface, \
-    ContainerSymbol
+    ContainerSymbol, TypeSymbol, RoutineSymbol
 from psyclone.psyir.symbols.datatypes import TypeSymbol, DeferredType
```

Comment:
Not yours but pylint says that `DeferredType` is unused.

Reply:
Done